bin/node/test-runner-example/src/lib.rs

```diff
@@ -71,6 +81,89 @@ impl ChainInfo for NodeTemplateChainInfo {
 		)
 	}
 
+	fn config(task_executor: TaskExecutor) -> Configuration {
```

Comment:
why can't this stay in `utils/test-runner`? I was happy that in the previous PR you made it such, but why is it being reverted now? 

I still think that 80% of the things being configured here are pretty irrelevant to the end user/programmer, thus not a good API.

Reply:
At least we can provide different variations where you have `fn config()` as it is now, and a `config_default()` or something like this.

Reply:
There are cases where we'd like to run the test runner as a bin and provide it with command line args like a real node. (This is all simnet stuff), yeah i suppose a default would be nice 🤔